src/splitInteger.test.js

```diff
@@ -8,12 +8,32 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger(10, 5)).toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    const a = 2;
+
+    expect(splitInteger(a, 1)).toEqual([2]);
+  });
+
+  it(`should return an integer array`, () => {
+    const a = (splitInteger(20, 4));
+
+    for (const b of a) {
+      expect(typeof b).toBe('number');
+    }
+  });
+
+  it(`should return an array with same lenghth as 'parts' value`, () => {
+    expect(splitInteger(20, 4).length).toBe(4);
+  });
```

Comment:
You can also use [toHaveLength](https://jestjs.io/docs/expect#tohavelengthnumber) here

Reply:
> You can also use [toHaveLength](https://jestjs.io/docs/expect#tohavelengthnumber) here

thanks, done!